testbed/tests/brakelight_test.py

```diff
@@ -0,0 +1,46 @@
+# firmware/testbed/tests/brakelight_test.py
+
+import time
+from testbeds.hil_testbed import teststand
+from drivers.common_drivers.can_driver import HILBoard
+from tqdm import tqdm
+
+# Define constants for the threshold voltage when the brake light turns on
+BRAKE_LIGHT_ON_THRESHOLD_VOLTAGE_1 = 833 # Lower threshold voltage
+BRAKE_LIGHT_ON_THRESHOLD_VOLTAGE_2 = 841 # Upper threshold voltage
+
+def test_brakelight(teststand):
+    print(teststand.vehicle_boards)
+    
+    vcu_hil: HILBoard = teststand.hil_boards["vcu_hil"]
+    # update pud -> pdu_hil
+    pdu_hil: HILBoard = teststand.hil_boards["pdu_hil"]
+
+    # Iterate over the entire voltage range
+    for voltage in tqdm(range(0, 3301)):
```

Comment:
is there a reason we are importing this tqdm library instead of just using a normal for loop with a step?